### PR TITLE
Fix the failure tests when building with JDK 11

### LINKED ISSUE #12587: Some failure tests when building with JDK 11
Following #12578

### PR DESCRIPTION
**Pull request by zhfeng**
Fixes #12587

### CODE REVIEW

File: shardingsphere-jdbc/shardingsphere-jdbc-core/src/test/java/org/apache/shardingsphere/driver/jdbc/adapter/invocation/JdbcMethodInvocationTest.java [hunk 1]
```diff
@@ -29,7 +29,22 @@ public void assertInvokeSuccess() throws NoSuchMethodException {
     
     @Test(expected = IllegalAccessException.class)
     public void assertInvokeFailure() throws NoSuchMethodException {
-        JdbcMethodInvocation actual = new JdbcMethodInvocation(String.class.getDeclaredMethod("indexOfSupplementary", int.class, int.class), new Object[] {1, 1});
+        JdbcMethodInvocation actual;
+
```
Comment (tristaZero) [inline shardingsphere-jdbc/shardingsphere-jdbc-core/src/test/java/org/apache/shardingsphere/driver/jdbc/adapter/invocation/JdbcMethodInvocationTest.java:33]: Could you remove this redundant blank line？

File: shardingsphere-jdbc/shardingsphere-jdbc-core/src/test/java/org/apache/shardingsphere/driver/jdbc/adapter/invocation/JdbcMethodInvocationTest.java [hunk 2]
```diff
@@ -29,7 +29,22 @@ public void assertInvokeSuccess() throws NoSuchMethodException {
...
+        }
         actual.invoke("");
     }
+
+    private boolean isJDK8OrEarlier() {
+        String version = System.getProperty("java.specification.version");
+        if (version.startsWith("1.")) {
+            version = version.substring(2);
+        }
+
```
Comment (tristaZero) [inline shardingsphere-jdbc/shardingsphere-jdbc-core/src/test/java/org/apache/shardingsphere/driver/jdbc/adapter/invocation/JdbcMethodInvocationTest.java:47]: Same with the last comment. Thanks.

**Review by tristaZero — CHANGES_REQUESTED** [review]

**Comment by zhfeng** [comment]
Thanks @tristaZero and I just remove these lines. Also is it possible to add this in the source code style check ?

**Review by tristaZero — APPROVED** [review]
Sorry for my late reply. ;(
But, thanks for your work here. ;-)

**Pull request merged**